actpool/actpool.go

```diff
@@ -434,6 +434,10 @@ func (ap *actPool) Validate(ctx context.Context, selp *action.SealedEnvelope) er
 	return ap.validate(ctx, selp)
 }
 
+func (ap *actPool) ValidateWithState(ctx context.Context, selp *action.SealedEnvelope) error {
+	panic("should not call actPool.ValidateWithState")
```

Comment:
returning nil is fine, means nothing checked

Reply:
in the tx_container PR I think we discussed it is better to panic for functions not intended to be called? that's why did it here

Reply:
to be removed